download_and_install/download_and_install.md

```diff
@@ -0,0 +1,52 @@
+Download and Install
+====================
+
+Pre-built binaries
+------------------
+
+OLA is available pre-built on many platforms:
+
+### Debian/Ubuntu (including Raspberry Pi)
+
+Raspberry Pi users may want to see the [Raspberry Pi Quick-Start Guide](ola_on_raspberry_pi)
+
+Ubuntu users will need the `universe` repository enabled.
```

Comment:
Is there an easy command to do so? Or some link where it is explained? (Non-Ubuntu user here ;))

Reply:
Linked to Ubuntu docs on the matter.